### PR TITLE
README: update known issues and add inbound network ports

### PR DESCRIPTION
**Pull request by teor2345**
## Motivation

Zebra's readme doesn't include the latest known issues or network features.

## Solution

- Update the known issues
- Add inbound network details

## Review

Anyone can review this PR. It is not urgent.

### Reviewer Checklist

  - [ ] Known issues are current
  - [ ] Network config is correct

## Related Issues

The network docs should have been updated as part of #2277, #2276, or previous related tickets.

## Follow Up Work

 Update the README goals and scope for NU5 activation #2372

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,11 +128,16 @@ Zebra primarily depends on pure Rust crates, and some Rust/C++ crates:
 ### Known Issues
 
 There are a few bugs in Zebra that we're still working on fixing:
-- [Peer connections sometimes fail permanently #1435](https://github.com/ZcashFoundation/zebra/issues/1435)
-  - these permanent failures can happen after a network disconnection, sleep, or individual peer disconnections
-  - workaround: use `Control-C` to exit `zebrad`, and then restart `zebrad`
+- [In rare cases, Zebra panics on shutdown](https://github.com/ZcashFoundation/zebra/issues/1678)
+  - For examples, see [#2055](https://github.com/ZcashFoundation/zebra/issues/2055) and [#2209](https://github.com/ZcashFoundation/zebra/issues/2209)
+  - These panics can be ignored, unless they happen frequently
+- [Interrupt handler does not work when a blocking task is running](https://github.com/ZcashFoundation/zebra/issues/1351)
```
Comment (teor2345) [inline README.md]:
```suggestion
- [Interrupt handler does not work when a blocking task is running #1351](https://github.com/ZcashFoundation/zebra/issues/1351)
```

File: README.md [hunk 2]
```diff
@@ -128,11 +128,16 @@ Zebra primarily depends on pure Rust crates, and some Rust/C++ crates:
 ### Known Issues
 
 There are a few bugs in Zebra that we're still working on fixing:
-- [Peer connections sometimes fail permanently #1435](https://github.com/ZcashFoundation/zebra/issues/1435)
-  - these permanent failures can happen after a network disconnection, sleep, or individual peer disconnections
-  - workaround: use `Control-C` to exit `zebrad`, and then restart `zebrad`
+- [In rare cases, Zebra panics on shutdown](https://github.com/ZcashFoundation/zebra/issues/1678)
```
Comment (teor2345) [inline README.md]:
```suggestion
- [In rare cases, Zebra panics on shutdown #1678](https://github.com/ZcashFoundation/zebra/issues/1678)
```

File: README.md [hunk 3]
```diff
@@ -82,7 +82,18 @@ We usually run `zebrad` on systems with:
 `zebrad` might build and run fine on smaller and slower systems - we haven't
 tested its exact limits yet.
 
-### Network Usage
+### Network Ports and Usage
```
Comment (teor2345) [inline README.md]:
```suggestion
### Network Ports and Data Usage
```

**Review by oxarbitrage — APPROVED** [review]
Looks good, thanks.

**Pull request merged**